docs/components/connectivity/wifi.md

```diff
@@ -1,4 +1,103 @@
 # WiFi
 
-!!! warning
-    This page is still in progress. You can check the description of blocks for help.
+## Overview
+
+The **WiFi** component can be used to manage the WiFi settings as well as to retrieve information about device's WiFi.
+
+## Methods
+
+### BSSID
+![](/assets/images/components/connectivity/wifi/m_bssid.png)
+
+\[None\]
+
+Returns the basic service set identifier (BSSID) of the current access point.
+
+### Disable
+![](/assets/images/components/connectivity/wifi/m_disable.png)
+
+\[ None \]  
+
+Disable the Wi-Fi
```

Comment:
Method descriptions should be in the format: "This method [description]", where the text "This method" is omitted.
In this case, the desc should be "This method **Disables the WiFi**"